app/index.html

```
@@ -627,6 +641,17 @@ <h2 class="card-title">Monthly Expenses</h2>
                                 </div>
                             </div>
 
+                            <!-- ENS / Crypto Wallet Lookup -->
+                            <div class="card glass-card">
+                                <h2 class="card-title">Crypto Wallet Lookup (ENS)</h2>
+                                <p class="card-subtitle">Resolve a .eth address and total the value held across every configured EVM chain.</p>
+                                <form class="ens-lookup-form" id="form-ens-lookup">
+                                    <input type="text" id="ens-lookup-input" placeholder="vitalik.eth" autocomplete="off">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)